deltachat-rpc-server/src/main.rs

```diff
@@ -11,6 +11,11 @@ use deltachat_jsonrpc::api::events::event_to_json_rpc_notification;
 use deltachat_jsonrpc::api::{Accounts, CommandApi};
 use futures_lite::stream::StreamExt;
 use tokio::io::{self, AsyncBufReadExt, BufReader};
+
+// See the comment for sigterm_task below.
+// #[cfg(target_family = "unix")]
+// use tokio::signal::unix as signal_unix;
```

Comment:
why are you using this instead of `tokio::signal::ctrl_c` which doesn't need the `cfg`?

Reply:
:scream: ugh, you're totally right.  the ctrlc crate has a feature to behave sanely in this regard, but tokio ignores it.  sad.